src/data_management/rasters/NewRaster2.js

```diff
@@ -26,6 +27,19 @@ class NewRaster2 extends Component {
   render() {
     return (
       <ManagementForm onSubmit={onSubmitExample} wizardStyle>
+        <Slushbucket
+          name="sharedWithOrganisation"
+          title="Shared With Organisation"
+          choices={[
+            { value: "abc", display: "aBc" },
+            { value: "nens", display: "nEns" },
+            { value: "home", display: "hOmbre" },
+          ]}
+          readonly={false}
```

Comment:
Ik heb bij de rest de readonly property weggedaan, bij disabled velden laat hij het veld helemaal niet zien. Twijfel of we het moeten hebben.

Reply:
Voor nu besloten readonly te laten bestaan.